test_runner/regress/test_sharding.py

```diff
@@ -1508,6 +1508,7 @@ def finish_split():
     env.storage_controller.consistency_check()
 
 
+@pytest.mark.skip(reason="The backpressure change has not been merged yet.")
```

Comment:
Not sure which backpressure we are talking about here. I don't see much point committing a test that is skipped.